codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/rds/AuthTokenGeneratorIntegration.kt

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
This is the new codegen integration that writes a new file with wrapper token generator if service is RDS.